pkg/operator/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go

```diff
@@ -241,7 +251,8 @@ func (n *klusterletController) sync(ctx context.Context, controllerContext facto
 	if klusterlet.Spec.RegistrationConfiguration != nil {
 		registrationFeatureGates = klusterlet.Spec.RegistrationConfiguration.FeatureGates
 		config.ClientCertExpirationSeconds = klusterlet.Spec.RegistrationConfiguration.ClientCertExpirationSeconds
-
+		config.RegistrationKubeAPIQPS = float32(klusterlet.Spec.RegistrationConfiguration.KubeAPIQPS)
```

Comment:
what if it is not set? it could happen during upgrade.

Reply:
will use the default values which are set at the beginning : https://github.com/open-cluster-management-io/ocm/pull/344/files#diff-cb893b14481c5389fd18eb242570f35dfc9ddccb794406ba79cf256c88d8a51dR193

Reply:
but the klusterlet.Spec.RegistrationConfiguration.KubeAPIQPS can be 0 for old klusterlet

Reply:
add flags for the agent like, will not set qps/burst flags if kubeAPIQPS is 0.
```
          {{if gt .RegistrationKubeAPIQPS 0.0}}
          - "--kube-api-qps={{ .RegistrationKubeAPIQPS }}"
          {{end}}
          {{if gt .RegistrationKubeAPIBurst 0}}
          - "--kube-api-burst={{ .RegistrationKubeAPIBurst }}"
          {{end}}
```